### ISSUE TITLE
bug: MD054 incorrectly flags allowed HTML tags when `autolink = false`

### ISSUE THREAD

**Issue by martimlobao** [rvben/rumdl#222]
False positive for MD054 when detecting autolink links. (Related to #221)

```toml
[global]
enable = ["MD033", "MD054"]

[MD033]
allowed-elements = ["br"]  # List of allowed HTML tags (default: none)

[MD054]
autolink = false    # Allow <https://example.com>
inline = true      # Allow [text](url) and ![alt](url)
full = true        # Allow [text][ref] with separate [ref]: url definition
collapsed = true   # Allow [text][] with separate [text]: url definition
shortcut = true    # Allow [text] with separate [text]: url definition
url-inline = true  # Allow [https://example.com](https://example.com)
```

```md
# Example

One line.<br>
Second line.
```

```
❯ rumdl check example.md
example.md:3:10: [MD054] Link/image style 'autolink' is not allowed
```

Apologies for the barrage of bug reports @rvben 😅

**Comment by rvben** [comment]
Fixed in v0.0.200. Thanks for reporting @martimlobao!
